app.json

```diff
@@ -122,6 +122,7 @@
           "image": "./assets/icon.png"
         }
       ],
+      "@maplibre/maplibre-react-native",
```

Comment:
Is it necessary to create a new build to test this package?

Reply:
yes